python/lsst/meas/astrom/anetAstrometry.py

```diff
@@ -48,6 +48,11 @@ class ANetAstrometryConfig(pexConfig.Config):
     rejectIter = pexConfig.RangeField(dtype=int, default=3, doc="Rejection iterations for Wcs fitting",
                                       min=0)
 
+    @property
+    def refObjLoader(self):
+        """An alias, for a uniform interface with the standard AstrometryTask"""
+        return self.solver
+
```

Comment:
I'm confused why we haven't seen this before.  Is it because no one uses this class, or did you need to do something special?